### PR TITLE
Add timestamp prior to signing

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Update code
* [x] Clean-up expected type of last_modified (eg. `'"42"'` or `'42'`) --> `'42'`
* [x] Update demo
* [x] Update scripts

Ref https://github.com/Kinto/kinto/issues/687

### CODE REVIEW

**Comment by leplatrem** [comment]
Ready to review!

@Natim @almet r?

Related:
* https://github.com/Kinto/kinto-http.js/pull/110
* https://github.com/Kinto/kinto-http.py/issues/77

File: README.rst [hunk 1]
```diff
@@ -59,6 +59,8 @@ Notes on canonical JSON
 
 Specific to Kinto:
 
+* The payload to be signature has two attributes: ``last_modified`` with the
```
Comment (almet) [inline README.rst]: Typo

Comment (almet) [inline README.rst]: Nit as a string

File: kinto_signer/serializer.py [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
-def canonical_json(records):
+def canonical_json(records, last_modified):
     records = copy.deepcopy(records)
     records = filter(lambda r: r.get('deleted', False) is not True, records)
     records = sorted(records, key=operator.itemgetter('id'))
 
-    return json.dumps(records, sort_keys=True, separators=(',', ':'))
+    payload = {'data': records, 'last_modified': '%s' % last_modified}
+
+    return json.dumps(payload, sort_keys=True, separators=(',', ':'))
```
Comment (almet) [inline kinto_signer/serializer.py]: I am not sure this is recursive

Comment (Natim) [inline kinto_signer/serializer.py]: Yes it is we validated it when building canonical JSON. I think there is even a test in this code base.

**Comment by Natim** [comment]
r+

**Comment by almet** [comment]
Really cool :)

**Pull request merged**